sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Connect-SPOService.md

```diff
@@ -19,6 +19,13 @@ This cmdlet must be run before any other SharePoint Online cmdlets can run.
 
 ## SYNTAX
 
+### AuthenticationCertificate
```

Comment:
these paramserternames needs to match the backend code. please have Siddhant provide you with an updated platyPS template.